net/ariang/Makefile

```diff
@@ -59,7 +59,22 @@ define Package/ariang/install
 		$(1)/www/ariang
 endef
 
-Package/ariang-nginx/install = $(Package/ariang/install)
+define Package/ariang-nginx/install
+	$(INSTALL_DIR) \
+		$(1)/www/ariang
+
+	$(CP) \
+		$(PKG_BUILD_DIR)/{langs,css,js,fonts} \
+		$(PKG_BUILD_DIR)/index.html \
+		$(PKG_BUILD_DIR)/index.manifest \
+		$(PKG_BUILD_DIR)/LICENSE \
+		$(PKG_BUILD_DIR)/favicon.ico \
+		$(1)/www/ariang
+	$(INSTALL_DIR) $(1)/etc/nginx
```

Comment:
I think that you could still avoid code duplication for the early part by referencing Package/ariang/install first andf then have the four new lines